### ISSUE TITLE
`gather_()` was deprecated in tidyr 1.2.0.

### ISSUE THREAD

**Issue by jkylearmstrong-temple** [kassambara/survminer#656]
````{R}
> ggcoxdiagnostics(res.cox)
Warning: `gather_()` was deprecated in tidyr 1.2.0.
ℹ Please use `gather()` instead.
ℹ The deprecated feature was likely used in the survminer package.
  Please report the issue at <https://github.com/kassambara/survminer/issues>.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated.
`geom_smooth()` using formula = 'y ~ x'
Warning: Failed to fit group -1.
Caused by error in `predLoess()`:
````

**Comment by BrennanHickson** [comment]
I received the same warning message.

Backtrace:
    ▆
 1. ├─tibble::tibble(surv_summary(cxsf0, data = complete_cases_all))
 2. │ └─tibble:::tibble_quos(xs, .rows, .name_repair)
 3. │   └─rlang::eval_tidy(xs[[j]], mask)
 4. └─survminer::surv_summary(cxsf0, data = complete_cases_all)
 5.   ├─base::cbind(...)
 6.   │ └─base::cbind(deparse.level, ...)
 7.   │   └─base::data.frame(..., check.names = FALSE)
 8.   └─survminer:::.flat(x$surv)
 9.     └─tidyr::gather_(x, key_col = "key", value_col = "value", gather_cols = colnames(x))

**Comment by kassambara** [comment]
fixed in #579 and #665
